modules/op-using-a-custom-database-in-tekton-hub.adoc

```diff
@@ -0,0 +1,47 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="op-using-a-custom-database-in-tekton-hub_{context}"]
+= Using a custom database in {tekton-hub}
+
+[role="_abstract"]
+After installation of {tekton-hub} on a cluster, administrators can use a custom database instead of the default PostgreSQL database installed by the Operator.
+
+[discrete]
+.Procedure
+
+. Create a secret in the `openshift-pipelines` namespace with the following keys: 
+* `host`
+* `port`
+* `username`
+* `password`
+* `database`
+
+. In the `TektonHub` CR, set the value of the `databaseCredentials` attribute to the custom database secret.
++
+.Example of custom categories 
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonHub
+metadata:
+  name: hub
+spec:
+  databaseCredentials: myExternalDatabaseSecret
+----
+
+. Migrate data from the default database to the custom database.
++
+[source,terminal]
+----
+$ 
+----
+
+. Delete the default database.
++
+[source,terminal]
+----
+$ oc delete 
```

Comment:
@PuneetPunamiya what is the command here?

Reply:
If the user is not using the default database and is using his own database then the default database is deleted by the operator itself, user don't need to delete the database in that case